tests/handlers/test_destinations.py

```diff
@@ -97,3 +99,67 @@ def test_post(self):
         d = NotificationDestination.query.get(d.id)
         self.assertEqual(d.name, data["name"])
         self.assertEqual(d.options["url"], data["options"]["url"])
+
+
+def test_discord_notify_calls_requests_post():
+    alert = mock.Mock(spec_set=["id", "name", "options", "render_template"])
+    alert.id = 1
+    alert.name = "Test Alert"
+    alert.options = {
+        "custom_subject": "Test custom subject",
+        "custom_body": "Test custom body",
+    }
+    alert.render_template = mock.Mock(return_value={"Rendered": "template"})
+    query = mock.Mock()
+    query.id = 1
+
+    user = mock.Mock()
+    app = mock.Mock()
+    host = "https://localhost:5000"
+    options = {
+        "url": "https://discordapp.com/api/webhooks/test"
+    }
+
+    new_state = "triggered"
```

Comment:
Use `Alert.TRIGGERED_STATE` instead of it.

Reply:
Done